docs/blog/posts/instance-volumes.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Introducing instance volumes to persist data on instances"
+date: 2024-11-05
+description: "To simplify caching across runs and the use of NFS, we introduce a new volume type that persists data on the instance."
+image: https://github.com/dstackai/static-assets/blob/main/static-assets/images/dstack-instance-volumes.png?raw=true
+slug: instance-volumes
+---
+
+# Introducing instance volumes to persist data on instances
+
+## How it works { style="display:none" }
+
+Until now, `dstack` supported data persistence only with network volumes, managed by clouds.
+While convenient, sometimes you might want to use a simple cache on the instance or 
+mount an NFS share to your SSH fleet. To address this, we're now introducing instance volumes that work for both cases.
+
+<div editor-title="examples/misc/volumes/cache.dstack.yml"> 
+    
+```yaml 
+type: task 
+name: llama32-task
+
+env:
+  - HF_TOKEN
+  - MODEL_ID=meta-llama/Llama-3.2-3B-Instruct
+commands:
+  - pip install vllm
+  - vllm serve $MODEL_ID --max-model-len 4096
+ports: [8000]
+
+volumes:
+  - /root/.dstack/cache:/root/.cache/pip
```

Comment:
What about model cache (`/root/.cache/huggingface/hub`)? I suggest either mount the whole `/root/.cache` container dir:

```yaml
volumes:
  - /root/.dstack/cache:/root/.cache
```

or use two instance volume mounts:

```yaml
volumes:
  - /root/.dstack/cache/pip:/root/.cache/pip
  - /root/.dstack/cache/huggingface:/root/.cache/huggingface
```